CONTRIBUTING.md

```diff
@@ -183,10 +183,10 @@ bundle exec rake test
 ```
 
 Sometimes getting the tests running can be too much work, especially if you don't have much Ruby experience.
-It's okay: be lazy and let our build bot [Travis](https://travis-ci.org/#!/github/linguist) run the tests for you.
+It's okay: be lazy and let GitHub Actions run the tests for you.
```

Comment:
Might be worth linking `GitHub Actions` to https://github.com/features/actions in case the reader's unfamiliar with Actions.